ludwig/features/vector_feature.py

```diff
@@ -102,26 +97,26 @@ def add_feature_data(
             raise
 
         # Determine vector size
-        vector_size = backend.df_engine.compute(
+        output_size = backend.df_engine.compute(
```

Comment:
I'm not 100% sold on this choice of renaming. The way I see it `vector_size` as a name works well for bot input and output features, while `output_size` doesn't really work for the input features. Maybe I'm missing something though.

Reply:
Hmm I see what you mean. The issue on the output vector features that requires renaming is that these parameters are passed directly to the decoder object (Projector class in this case) also used by other features. I'll think about how to fix this and upload another patch.

Reply:
Resolved offline